### DISCUSSION TITLE
PLEASE HELP

### DISCUSSION THREAD

**Discussion by angelplays1936** [FabricMC/fabric-api#3105] [Player support]
Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'Fabric Loader' (fabricloader) 0.14.8 with any version between 0.14.19 (inclusive) and 0.15- (exclusive).
Unmet dependency listing:
	 - Mod 'Fabric API' (fabric-api) 0.76.0+1.18.2 requires version 0.14.9 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Expanded Storage' (expandedstorage) 8.3.3 requires version 0.14.9 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Pehkui' (pehkui) 3.7.5+1.14.4-1.20 requires version 0.14.19 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Mythic Mounts' (mythicmounts) 1.18.2-7.1 requires version 0.14.10 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Kanos Config' (kanos_config) 0.4.1+1.14.4-1.19.4 requires version 0.14.17 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Steel' (steel) 1.3.0+1.18.2 requires version 0.14.9 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Vanity Slots' (vanityslots) 1.2.8 requires version 0.14.11 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	 - Mod 'Fabric Block API (v1)' (fabric-block-api-v1) 1.0.0+ee3621fc60 requires version 0.14.9 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.14.8!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

**Comment by Poopooracoocoo** [comment]
You need to update Fabric Loader. This is done differently based on the launcher you have. If you're using Mojang's Minecraft Launcher, follow the [Fabric Wiki](https://fabricmc.net/wiki/player:tutorials:update_mcl). Note that Fabric _Loader_ works for all Minecraft versions. 

I believe this prompt may have been improved in newer versions of Fabric Loader...
